runtime/docker/docker.go

```diff
@@ -256,7 +256,33 @@ func (d *DockerRuntime) CreateNet(ctx context.Context) (err error) {
 	// this was added to allow mgmt network gw ip to be available in a startup config templation step (ceos)
 	var v4, v6 string
 	if v4, v6, err = utils.FirstLinkIPs(bridgeName); err != nil {
```

Comment:
@carlmontanari was it painful to look at that err handling line? :P

Reply:
lol yes! im too dumb to see it all on the one line!!!!